### PR TITLE
GlobalDelight.Boom3D version 1.3.4

### PR DESCRIPTION
**Pull request by AlexDev404**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/62107)

### CODE REVIEW

**Comment by AlexDev404** [comment]
### Package Fails During Install - Dependency Issues
Requires: DotNet 4

**Comment by AlexDev404** [comment]
https://github.com/microsoft/winget-pkgs/pull/62107#issuecomment-1138901699

**Comment by ItzLevvie** [comment]
cc @denelon: Add "Blocking-Issue" and "Dependencies"

**Review by ItzLevvie — APPROVED** [review]

**Comment by AlexDev404** [comment]
Alright, no problem!

**Comment by denelon** [comment]
Hey @AlexDev404, we're now doing validation with dependencies if you want to add them to the manifest :)

**Comment by stephengillie** [comment]
> Url validation error
> 
> * manifests\g\GlobalDelight\Boom3D\1.3.4
>   
>   * https://www.boom3dapp.com/legal/LaFab/boom-for-mobile-tc.pdf
>     
>     * Error Message: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. ([www.boom3dapp.com:443](http://www.boom3dapp.com:443)) A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.

Verify:
image

File: manifests/g/GlobalDelight/Boom3D/1.3.4/GlobalDelight.Boom3D.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+PackageLocale: en-US
+Publisher: Global Delight
+PublisherUrl: https://www.globaldelight.com/
+PublisherSupportUrl: https://www.globaldelight.com/contactsupport
+PrivacyUrl: https://www.globaldelight.com/privacypolicy
+Author: Global Delight Technologies Pvt. Ltd.
+PackageName: Boom 3D
+PackageUrl: https://www.globaldelight.com/boom/
+License: Proprietary
+LicenseUrl: https://www.boom3dapp.com/legal/LaFab/boom-for-mobile-tc.pdf
```
Comment (stephengillie) [inline manifests/g/GlobalDelight/Boom3D/1.3.4/GlobalDelight.Boom3D.locale.en-US.yaml:15]:
```suggestion
# LicenseUrl: 
```

**Pull request closed without merging**